frontend/src/styles/global.scss

```diff
@@ -684,3 +684,23 @@ body {
         display: none;
     }
 }
+
+.line-clamp-2 {
+    overflow: hidden;
+    display: -webkit-box;
+    -webkit-box-orient: vertical;
+    -webkit-line-clamp: 2;
+}
+
+.object-cover {
```

Comment:
this should go in `utilities.scss`

and really we should loop over the options in https://tailwindcss.com/docs/object-fit to create them all